code_samples/live_templates/build.gradle

```diff
@@ -1,24 +1,27 @@
 // Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
 
 plugins {
-    id 'java'
-    id 'org.jetbrains.intellij' version '0.4.16'
+  id 'java'
+  id 'org.jetbrains.intellij' version '0.4.16'
 }
 
 group 'org.intellij.sdk'
-version '2.0.0'
+version '2.1.0'
 
 sourceCompatibility = 1.8
 
 repositories {
-    mavenCentral()
+  mavenCentral()
 }
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
-    version '2019.3.3'
-    updateSinceUntilBuild = false
+//    version '2020.1'
```

Comment:
I'm not sure what you mean by "no version". 
The default `intellij.version` is 'LATEST-EAP-SNAPSHOT', which is needed for these changes. This will have to be changed at 2020.1 release of course.

I can include `version 'LATEST-EAP-SNAPSHOT'` if that is clearer.

Reply:
`201-EAP-SNAPSHOT` would be more clear IMHO

Reply:
Works for me...